rick-morty-app-client/src/pages/404/NotFoundPage.tsx

```diff
@@ -1,10 +1,15 @@
+import { t } from "i18next";
 import React, { useEffect } from "react";
 import { Container, Button } from "react-bootstrap";
 import { useNavigate } from "react-router-dom";
 
+import { useTranslation } from "react-i18next";
+
 const NotFoundPage: React.FC = () => {
   const navigate = useNavigate();
 
+  const { t }: { t: (key: string) => string } = useTranslation();
```

Comment:
napewno musisz to t typuowac?,powinno to dzialac

Reply:
Jak robiłam bez typowania wyskakiwał mi błąd: "TS2554: Expected 0 arguments, but got 1.", 

dlatego zaczęłam typować to t.

Reply:
no i jak typuje to wszystko działa, sugeruje, aby to zostawić jak jest